doc/api/modules.md

```diff
@@ -233,6 +232,23 @@ This property is experimental and can change in the future. It should only be us
 by tools converting ES modules into CommonJS modules, following existing ecosystem
 conventions. Code authored directly in CommonJS should avoid depending on it.
 
+To create an ESM module that should provide an arbitrary value to CommonJS, the
+`__cjsModule: true` marker can be used instead:
```

Comment:
I think we should consider discussing this in a separate issue. It's not the first time that we run into the need to save library users a `loaded.default` pickup when they use a native loader. The same has been happening for `import d from 'esm_transpiled_to_cjs'` as well and seems to have already been causing great pain for years  - https://github.com/babel/babel/issues/12363 contains an great description of the problem. I think we should invent a marker that helps both transpiled ESM and real ESM indicate the intent to unwrap a single default export (and ideally, we should error when we found named exports along with this marker, because unwrapping breaks them). Something like `exports.unwrapSingleDefault = true`  in transpiled ESM and `export const unwrapSingleDefault = true`  in real ESM may suffice - but we also need to get the buy-in from transpilers to implement the first.

Reply:
Another alternative is to unwrap based on the presence of `__esModule`, but my hunch is the ship has already sailed and it is now too late to use this for unwrapping, as users of these transpiled packages may have already been relying on doing the unwrapping themselves.

Reply:
I've motivated this in a separate issue in https://github.com/nodejs/node/issues/54085 now.